bin/impl/setup-hadoop.sh

```diff
@@ -36,23 +36,36 @@ print_to_console "    * ResourceManager status: http://localhost:8088/"
 print_to_console "    * view logs at $HADOOP_LOG_DIR"
 
 rm -rf "$INSTALL"/hadoop-*
-rm -f "$HADOOP_LOG_DIR"/*
-rm -rf "$HADOOP_LOG_DIR"/application_*
+rm -rf "$HADOOP_LOG_DIR"/*
 rm -rf "$DATA_DIR"/hadoop
 mkdir -p "$HADOOP_LOG_DIR"
 
 tar xzf "$DOWNLOADS/$HADOOP_TARBALL" -C "$INSTALL"
 
 hadoop_conf="$HADOOP_HOME"/etc/hadoop
-cp "$UNO_HOME"/conf/hadoop/* "$hadoop_conf/"
+cp "$UNO_HOME"/conf/hadoop/core-site.xml "$hadoop_conf/"
```

Comment:
Could have `conf/hadoop-common`, `conf/hadoop2`, and `conf/hadoop3` dirs.  Then the script could do something like the following.

```bash
cp "$UNO_HOME"/conf/hadoop-common/* "$hadoop_conf/"
if [[ $HADOOP_VERSION =~ ^2\..*$ ]]; then
  cp "$UNO_HOME"/conf/hadoop2/* "$hadoop_conf/"
￼ else
  cp "$UNO_HOME"/conf/hadoop3/* "$hadoop_conf/"
￼ fi
```

This avoids hard coding the files in the script, which could allow the user to drop additional files in the dirs.

Reply:
This is a good idea. I will work on it and do it as another PR.  It can be done for Accumulo too.